docs/contributor/storage/default_csi_plugin.md

```diff
@@ -40,7 +40,11 @@ The user provided lvmd config should be written to the same directory as the Mic
 If there is only one volume group on the system, LVMS uses it by
 default. If there are multiple volume groups, and no configuration
 file, LVMS looks for a volume group named `microshift`. If there is no
-volume group named `microshift`, LVMS is disabled.
+volume group named `microshift`, LVMS is disabled.  MicroShift also checks
+for the presence of volume groups on the system via `vgs`. If none exist, then
```

Comment:
I feel like following sentence is misplaced - whole paragraph is about scanning volume groups.

> MicroShift also checks for the presence of volume groups on the system via `vgs`.

I'd move it to beginning of the paragraph and clearly separate 3 cases:
- no vg -> no topolvm
- 1 vg -> topolvm
- many vgs, one of them named microshift -> topolvm

Reply:
+1